lib/internal/errors.js

```diff
@@ -581,16 +582,17 @@ let MAX_STACK_MESSAGE;
  * @returns {boolean}
  */
 function isStackOverflowError(err) {
-  if (MAX_STACK_MESSAGE === undefined) {
+  if (MAX_STACK_ERROR_MESSAGE === undefined) {
     try {
       function overflowStack() { overflowStack(); }
       overflowStack();
     } catch (err) {
-      MAX_STACK_MESSAGE = err.message;
+      MAX_STACK_ERROR_MESSAGE = err.message;
+      MAX_STACK_ERROR_NAME = err.name;
     }
   }
 
-  return err.name === 'RangeError' && err.message === MAX_STACK_MESSAGE;
+  return err.name === MAX_STACK_ERROR_NAME && err.message === MAX_STACK_ERROR_MESSAGE;
```

Comment:
The linter failed because this line is now longer than 80 characters. You'll need to either split this across two lines or shorten the names of ALL_CAPS variables.

Especially since they're not constants, it may be more stylistically appropriate to camelCase them? That would shave off 6 characters, making the line exactly 80 characters long. Perfect!  :-D

Reply:
Thanks, should be fixed up now.

Reply:
@Trott Hmm, I'm not sure about this... They are constants -- just lazily loaded constants.